core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -1,119 +1,118 @@
 <div class="story-summary-tile protractor-test-story-summary-title">
   <div class="story-progress-bar">
-    <svg class="story-progress-svg" width="50" height="50" ng-if="$ctrl.nodeCount">
+    <svg class="story-progress-svg" width="50" height="50" *ngIf="nodeCount">
       <circle class="story-progress-circle total-circle"
               cx="25"
               cy="25"
               r="20"
-              stroke-dasharray="<[$ctrl.getStrokeDashArrayValues()]>"
+              [attr.stroke-dasharray]="getStrokeDashArrayValues()"
               stroke-dashoffset="-2.5"
               stroke-linecap="round"/>
       <circle class="story-progress-circle completed-circle"
               cx="25"
               cy="25"
               r="20"
-              stroke-dasharray="<[$ctrl.getCompletedStrokeDashArrayValues()]>"
+              [attr.stroke-dasharray]="getCompletedStrokeDashArrayValues()"
               stroke-dashoffset="-2.5"
-              stroke-linecap="<[$ctrl.completedStoriesCount ? 'round' : 'butt']>"/>
+              [attr.stroke-linecap]="completedStoriesCount ? 'round' : 'butt'"/>
       <text x="25"
             y="-20"
             text-anchor="middle"
             stroke="#000"
             stroke-width="0.5"
             font-size="0.9em"
             transform="rotate(90)">
-        <[$ctrl.storyProgress]>%
+        {{ storyProgress }}%
       </text>
     </svg>
   </div>
   <div class="oppia-story-title">
     <strong class="title-text">
-      <a ng-href="<[$ctrl.getStoryLink()]>" ng-class="{'disabled-link': !$ctrl.classroomUrlFragment}"><[$ctrl.storySummary.getTitle()]> </a>
+      <a [href]="getStoryLink()" [ngClass]="{'disabled-link': !classroomUrlFragment}">{{ storySummary.getTitle() }} </a>
     </strong>
     <span class="oppia-chapters-count">
-      <span translate="I18N_TOPIC_VIEWER_CHAPTERS"
-            translate-values="{count: $ctrl.nodeCount}"
-            translate-interpolation="messageformat">
```

Comment:
Couldn't find substitute for this. Individual i18n looks something like 
`    "I18N_TOPIC_VIEWER_CHAPTERS": "{count, plural, one{1 chapter} other{# chapters}}",`

Reply:
Have you tested the UI, is it working fine?

Reply:
![Screenshot from 2021-05-10 17-22-35](https://user-images.githubusercontent.com/60519303/117656867-7e88f080-b1b6-11eb-9757-9f9a6cd7c77a.png)
![Screenshot from 2021-05-10 17-36-28](https://user-images.githubusercontent.com/60519303/117656873-8183e100-b1b6-11eb-9ff0-02f11e24e993.png)
Yes working fine!

Reply:
@mridul-netizen Please undo this change.... For a lot of languages the plural form is different form what we see in english (Russian for example). Even though for this case it might not affect, this sets a precedent that just shouldn't be allowed. How about we wait #12790. It should go in soon. Then you won't have to do all this to get pluralisation working.

Reply:
done